### DISCUSSION TITLE
Do rainbowkit support Martian wallet( a Aptos wallet)?

### DISCUSSION THREAD

**Discussion by rubb1sh** [rainbow-me/rainbowkit#1275] [Q&A]
As I said in title, Do rainbowkit support Martian wallet or any aptos wallet?

**Comment by DanielSinclair** [comment]
dApps can add any chain including Aptos chain via Wagmi, or even [Custom Chains](https://www.rainbowkit.com/docs/custom-chains) and can also add a [Custom Wallet](https://www.rainbowkit.com/docs/custom-wallets) connector to support a particular wallet like Martian Wallet.
